lib/jekyll/commands/serve.rb

```diff
@@ -219,17 +223,11 @@ def webrick_opts(opts)
         #
 
         private
-        def start_up_webrick(opts, destination)
-          if opts["livereload"]
-            @reload_reactor.start(opts)
-          end
-
+        def setup_webrick(opts, destination)
```

Comment:
you can't change the method name (even if it is a `private` method) because it will break Jekyll Admin's monkey-patch which is against your primary goal with this PR

Reply:
Since I moved the location of processing such as boot and open browser,  I think that it is necessary to patch the jekyll-admin side in either case.

I am thinking to apply the following patch.

```ruby
class Serve < Command
  class << self
    def setup_webrick(opts, destination)
      super(opts, destination)
      jekyll_admin_monkey_patch
    end
    
    def jekyll_admin_monkey_patch
      @server.mount "/admin", Rack::Handler::WEBrick, JekyllAdmin::StaticServer
      @server.mount "/_api",  Rack::Handler::WEBrick, JekyllAdmin::Server
      Jekyll.logger.info "JekyllAdmin mode:", ENV["RACK_ENV"] || "production"
    end
  end
end
```